homeassistant/components/kostal_plenticore/select.py

```diff
@@ -0,0 +1,129 @@
+"""Platform for Kostal Plenticore select widgets."""
+from __future__ import annotations
+
+from abc import ABC
+from datetime import timedelta
+import logging
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DEVICE_DEFAULT_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, SELECT_SETTINGS_DATA
+from .helper import Plenticore, SelectDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
```

Comment:
Please type the whole signature when adding type annotations, ie type `async_add_entities`.

Reply:
Fixed in https://github.com/home-assistant/core/pull/59998